greenriver/hmis-warehouse
# Superset roles [devops 702]

## Description

Removes superset config which will live in the superset-sync repo. Provide list of role names to superset so it can determine the corresponding role on that side. 

How to test:
- Needs a forthcoming update to superset to be testable. Ask @yakloinsteak to know if it's ready.

On login, if
- your email ends in greenriver.(org|com), you should become a superset **Admin**
- you set the superset_role value to a valid superset role, you become that role in superset
- If you set nothing for the superset roles and your email isn't a greenriver email, you become a report dashboard reader, which only exists for M***. populating that role has not been automated yet.

## Type of change

## Checklist before requesting review
